command/agent/config_parse.go

```diff
@@ -65,6 +65,7 @@ func ParseConfigFile(path string) (*Config, error) {
 		Telemetry: &Telemetry{},
 		Vault:     &config.VaultConfig{},
 		Vaults:    map[string]*config.VaultConfig{},
+		Reporting: config.DefaultReporting(),
```

Comment:
```suggestion
		Reporting: &config.Reporting{},
```

It doesn't matter all that much, but for consistency we initialize this struct with zero values